ddcommon-ffi/src/endpoint.rs

```diff
@@ -49,5 +49,11 @@ pub extern "C" fn ddog_endpoint_from_api_key_and_site(
     None
 }
 
+#[no_mangle]
+#[must_use]
+pub extern "C" fn ddog_endpoint_empty() -> Box<Endpoint> {
+    Box::new(Endpoint::default())
+}
+
```

Comment:
I would not provide that, but just do nothing if an API key is provided in the Endpoint instead of handling an empty endpoint.